### PR TITLE
GEOMESA-1992 Pom changes for publishing to maven central

### PR DESCRIPTION
**Pull request by elahrvivaz**
* Use the 'central' profile to deploy to ossrh, then manually release from the sonatype UI

Signed-off-by: Emilio Lahr-Vivaz <elahrvivaz@ccri.com>

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -249,59 +257,104 @@
...
+                                <goals>
+                                    <goal>jar</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+            <distributionManagement>
                 <repository>
```
Comment (jnh5y) [inline pom.xml]: I wonder if this should remain...  How does this block get used by people/programs?

File: pom.xml [hunk 2]
```diff
@@ -2843,6 +2893,39 @@
         <tag>HEAD</tag>
     </scm>
 
+    <developers>
```
Comment (jnh5y) [inline pom.xml]: Did you copy this from the geomesa-geoserver bit?  (If so, great!)

Comment (elahrvivaz) [reply pom.xml]: yep :)

File: pom.xml (hunk 1 above)
Comment (elahrvivaz) [reply pom.xml]: if we want to publish to locationtech we should be able to use this profile to do so. or do you mean we should keep it in the default pom?

**Pull request merged**